internal/xds/v3/contour_test.go

```diff
@@ -81,7 +82,7 @@ func TestXDSHandlerStream(t *testing.T) {
 					}, nil
 				},
 			},
-			want: fmt.Errorf("proto: Marshal called with nil"),
+			want: protoimpl.X.NewError("invalid nil source message"),
```

Comment:
could also change the test to assert on error strings, so we don't have to fish out how exactly the error is constructed